lib/cocoapods/installer/xcode/pods_project_generator.rb

```diff
@@ -233,6 +228,10 @@ def wire_target_dependencies(project, target_installation_results)
                       test_native_target.add_dependency(test_resource_bundle_target)
                     end
                   end
+                  dependent_test_project = pod_target_installation_results_hash[test_dependent_target.name].native_target.project
```

Comment:
this is missing on app specs below.

Reply:
Nice catch. Updated with a spec as well.